subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/catalog/CatalogPluginsKotlinDSLIntegrationTest.groovy

```diff
@@ -22,13 +22,16 @@ import org.gradle.test.fixtures.file.LeaksFileHandles
 import org.gradle.test.fixtures.plugin.PluginBuilder
 import org.gradle.test.fixtures.server.http.MavenHttpPluginRepository
 import org.junit.Rule
+
 /**
  * Mirror test of {@link CatalogPluginsGroovyDSLIntegrationTest}.
  *
  * This test isn't meant to check the behavior of the extension generation like the other
  * integration tests in this package, but only what is very specific to the Kotlin DSL.
  * Because it requires the generated Gradle API it runs significantly slower than the other
  * tests so avoid adding tests here if they cannot be expressed with the Groovy DSL.
+ *
+ * These tests use Groovy settings file because the parent class sets up things in it already.
```

Comment:
💄 
```suggestion
 * These tests use the Groovy settings file because the parent class already sets up things in it.
```

Reply:
Good catch! Fixed differently by making it plural. It was a typo.